nextflow.config

```diff
@@ -19,6 +19,9 @@ params {
     // Illumination correction
     illumination               = null
 
+    // Background subtraction
+    backsub                    = false
```

Comment:
Oh, yes, that's a good idea. We could trigger background subtraction on the presence of a `remove` column in the marker sheet.
Is there a better column to use?

Reply:
The column critical for the module would be `background`, as only using the tool for channel removal seems like an unnecessary data duplication step.

Reply:
ok, sounds good, I'll use the `background` column. Thanks!
I thought it might be use for something else as well, but it seemed like the `remove` column would only be used for background subtraction.

Reply:
Just to add here, it might be good to check for both the presence of the `background` and `exposure` columns as both are required for channel subtraction. If you check by `background`, and no `exposure` column is given, the tool would give an error.

I think the best way to approach this would be to: check for presence of the `background` column, if it is given, check the values in the column, if none are a string, we skip Backsub. Otherwise, check for presence of the `exposure` column (and values), if not provided, an error should be thrown, and otherwise Backsub is run.

Reply:
I still need to work out how to remove params.backsub as requested at the top of this comment.

Reply:
reverted back to using params.backsub since removing it got complicated quickly, as discussed below.